desktop/extensions-sdk/build/set-up/backend-extension-tutorial.md

```diff
@@ -0,0 +1,419 @@
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	var socketPath string
+	flag.StringVar(&socketPath, "socket", "/run/guest/volumes-service.sock", "Unix domain socket to listen on")
+	flag.Parse()
+
+	os.RemoveAll(socketPath)
+
+	logrus.New().Infof("Starting listening on %s\n", socketPath)
+	router := echo.New()
+	router.HideBanner = true
+
+	startURL := ""
+
+	ln, err := listen(socketPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	router.Listener = ln
+
+	router.GET("/hello", hello)
+
+	log.Fatal(router.Start(startURL))
+}
+
+func listen(path string) (net.Listener, error) {
+	return net.Listen("unix", path)
+}
+
+func hello(ctx echo.Context) error {
+	return ctx.JSON(http.StatusOK, HTTPMessageBody{Message: "hello world"})
+}
+
+type HTTPMessageBody struct {
+	Message string
+}
+```
+
+  </div>
+  <div id="node-app" class="tab-pane fade" markdown="1">
+
+<br/>
+
+> **Important**
+>
+> We don't have a working example for Node yet. [Fill out the form](https://docs.google.com/forms/d/e/1FAIpQLSdxJDGFJl5oJ06rG7uqtw1rsSBZpUhv_s9HHtw80cytkh2X-Q/viewform?usp=pp_url&entry.25798127=Node)
+> and let us know you'd like a sample for Node.
+{: .important }
+
+  </div>
+  <div id="python-app" class="tab-pane fade" markdown="1">
+
+<br/>
+
+> **Important**
+>
+> We don't have a working example for Python yet. [Fill out the form](https://docs.google.com/forms/d/e/1FAIpQLSdxJDGFJl5oJ06rG7uqtw1rsSBZpUhv_s9HHtw80cytkh2X-Q/viewform?usp=pp_url&entry.25798127=Python)
+> and let us know you'd like a sample for Python.
+{: .important }
+
+  </div>
+  <div id="java-app" class="tab-pane fade" markdown="1">
+
+<br/>
+
+> **Important**
+>
+> We don't have a working example for Java yet. [Fill out the form](https://docs.google.com/forms/d/e/1FAIpQLSdxJDGFJl5oJ06rG7uqtw1rsSBZpUhv_s9HHtw80cytkh2X-Q/viewform?usp=pp_url&entry.25798127=Java)
+> and let us know you'd like a sample for Java.
+{: .important }
+
+  </div>
+  <div id="net-app" class="tab-pane fade" markdown="1">
+
+<br/>
+
+> **Important**
+>
+> We don't have a working example for .Net. [Fill out the form](https://docs.google.com/forms/d/e/1FAIpQLSdxJDGFJl5oJ06rG7uqtw1rsSBZpUhv_s9HHtw80cytkh2X-Q/viewform?usp=pp_url&entry.25798127=.Net)
+> and let us know you'd like a sample for .Net.
+{: .important }
+
+  </div>
+</div>
+
+## Adapt the Dockerfile
+
+> **Note**
+>
+> When using the `docker extension init`, it creates a `Dockerfile` that already contains what is needed for a Go backend.
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#go-dockerfile" data-group="go">For Go</a></li>
+  <li><a data-toggle="tab" data-target="#node-dockerfile" data-group="node">For Node</a></li>
+  <li><a data-toggle="tab" data-target="#python-dockerfile" data-group="python">For Python</a></li>
+  <li><a data-toggle="tab" data-target="#java-dockerfile" data-group="java">For Java</a></li>
+  <li><a data-toggle="tab" data-target="#net-dockerfile" data-group="net">.For Net</a></li>
+</ul>
```

Comment:
Something is broken in this tab selection, it navigates to a different group of tabs

Reply:
@aevesdocker there is a weird behavior, I can show you on zoom if you want. But nothing blocking for us RN :)